provisioning/kubernetes/templates/templates-api.yaml

```diff
@@ -78,6 +78,16 @@ spec:
                 - name: Host
                   value: KubernetesReadinessProbe
             periodSeconds: 10
+            failureThreshold: 1
```

Comment:
`failureThreshold: 1` explicitne som uviedol, ze ak pod neodpoveda, tak sa ma hned odpojit (ale to nesposobi restart).